com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -21,7 +22,75 @@ public static function create($params) {
     return $instance;
   }
 
+  /**
+   * Get options for a given job roles field along with their database IDs.
+   *
+   * @param String $fieldName
+   *
+   * @return Array
+   */
+  public static function buildDbOptions($fieldName) {
+    $queryParam = array(1 => array($fieldName, 'String'));
+    $query = "SELECT t1.id, t1.label from civicrm_option_value t1
+              LEFT JOIN civicrm_option_group t2 on t1.option_group_id = t2.id
+              WHERE t2.name = %1";
+    $options = [];
+    $result = CRM_Core_DAO::executeQuery($query, $queryParam);
+    while ($result->fetch()) {
+      $options[] =  [ 'id'=>$result->id, 'label'=>strtolower($result->label) ];
```

Comment:
Please can you change `[ 'id'=>$result->id, 'label'=>strtolower($result->label) ]` to `array()` syntax because of php version compaitibility.